src/CMakeLists.txt

```diff
@@ -34,10 +34,14 @@ target_link_libraries(ptxview ${PTEX_LIBRARIES}
                               ${GLUT_LIBRARIES}
                               ${OPENGL_LIBRARIES}
                               ${ZLIB_LIBRARIES})
+IF(WIN32)
+  target_link_libraries(ptxview ${GLEW_LIBRARIES})
+ENDIF(WIN32)
```

Comment:
I'm not generally a fan of having additional dependencies on a per-platform basis.

It's hard to know why we would want to use GLEW based on just the code changes and the commit message. Can you please explain why we need to use GLEW?

If your justification for GLEW is valid then it might apply to Linux as well. In that scenario we would probably prefer to use GLEW across the board rather than limiting it to just WIN32.

It's hard to make that call without more details, though, so I would recommend adding these details to relevant comments and the commit message.

Reply:
I have tested GLEW compiling successfully on Ubuntu. Platform constraints have been removed in the new code submission